src/online2bin/online2-mic-asr-demo.cc

```diff
@@ -0,0 +1,303 @@
+// online2bin/online2-mic-asr-demo.cc
+
+// Copyright 2014  Johns Hopkins University (author: Daniel Povey)
```

Comment:
This online demo is potentially an interesting project but I'l like to separate it from the rest of the aishell2 recipe.
I am a little cautious, in general, about these types of demos because they tend to be the cause of a lot of installation difficulties and also tend to be hard to maintain; and they potentially are a distraction from the core parts of Kaldi, attracting what I consider to be the wrong kind of user- i.e. clueless users doing small-scale projects.
But it's nice to have something that works on mac because a lot of people have macs and they are sufficiently homogeneous that installation difficulties are less likely than they would be on linux or windows.  So I'll definitely consider this-- as a separate pull request after the main recipe is merged.